src/components/entity/download-button.vue

```diff
@@ -11,17 +11,14 @@ except according to the terms contained in the LICENSE file.
 -->
 <template>
   <a id="entity-download-button" class="btn btn-primary" :href="href">
-    <span class="icon-arrow-circle-down"></span>{{ text }}
+    <span class="icon-arrow-circle-down"></span>
+    <span>{{ $t('action.download') }}</span>
   </a>
 </template>
 
 <script setup>
 import { computed, inject } from 'vue';
-import { useI18n } from 'vue-i18n';
-
 import { apiPaths } from '../../util/request';
-import { useI18nUtils } from '../../util/i18n';
-import { useRequestData } from '../../request-data';
 
 const props = defineProps({
```

Comment:
It doesn't look like `DatasetEntities` passes an `odataFilter` prop to this component anymore. If the component doesn't use the prop anymore, it could remove the prop. But it sounds to me from the release criteria that the component does need the prop:

> - Except if I have an active filter then above the checkboxes I should see a dropdown with two options:
>   - Download # submissions matching the filter
>   - Download all # submissions